src/XMakeBuildEngine/UnitTests/Construction/ElementLocation_Tests.cs

```diff
@@ -112,7 +112,7 @@ public void TestLargeElementLocationUsedLargeColumn()
             }
             catch (InvalidProjectFileException ex)
             {
-                Assert.Equal(70012, ex.ColumnNumber);
+                Assert.Equal(1, ex.ColumnNumber);
```

Comment:
It sure seems like it--we're deliberately inserting 70k spaces and no newline, so I'd expect that element to start around column 70k.